### DISCUSSION TITLE
Welcome to frontend-hard-mode-interview Discussions!

### DISCUSSION THREAD

**Discussion by coffe1891** [coffe1891/frontend-hard-mode-interview#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by keenesun** [comment]
HI，Mr. Author，thank you for your contribution! But I am confused about how to strengthen comprehension which we can't apply in our company, could you give some advice, please?

>
